### PR TITLE
MAM-1988-exclude-outdated-follow-recommendations

### PR DESCRIPTION
**Pull request by NathanRLiu**
😎😎😎

### CODE REVIEW

File: app/controllers/api/v2/follow_recommendations_graph_controller.rb [hunk 1]
```diff
@@ -14,6 +15,8 @@ def show
     recommendations = recommendation_handles
                       .reject { |recommendation| follows.include?(recommendation) }
                       .filter_map { |h| handle_to_account_remote(h) }
+                      .reject { |recommendation| recommendation.last_status_at.nil? }
+                      .reject { |recommendation| recommendation.last_status_at < Time.zone.today - INACTIVITY_DAYS_CUTOFF }
                       .take(limit_param(RECOMMENDED_ACCOUNTS_LIMIT))
```
Comment (jtomchak) [inline app/controllers/api/v2/follow_recommendations_graph_controller.rb:20]: Do the 'last_status' date come as a Date Timezone? or just a standard ISO Format  "2023-07-13T13:30:42.719Z". 
If it's not a timezone formatted date then maybe something like:
```
INACTIVITY_DAYS_CUTOFF = Date.today - 30.days

...

recommendation.last_status_at < INACTIVITY_DAYS_CUTOFF

```

**Review by jtomchak — CHANGES_REQUESTED** [review]
Just a question about the incoming format of the `last_status_at` date. Otherwise the reject chaining is awesome!

File: app/controllers/api/v2/follow_recommendations_graph_controller.rb (hunk 1 above)
Comment (NathanRLiu) [reply app/controllers/api/v2/follow_recommendations_graph_controller.rb:20]: @jtomchak It is in ISO format.

Comment (jtomchak) [reply app/controllers/api/v2/follow_recommendations_graph_controller.rb:20]: excellent. thanks for checking

**Review by jtomchak — APPROVED** [review]
well done.

**Comment by jtomchak** [comment]
Exclude outdated follow recommendations

**Pull request merged**